react/Product.js

```diff
@@ -157,7 +157,7 @@ export const parseToJsonLD = ({
   const productLD = {
     '@context': 'https://schema.org/',
     '@type': 'Product',
-    '@id': product.link,
+    '@id': `/${product.linkText}/p`,
```

Comment:
Can you share where you learn that the link should be the path, not the whole URL?
I was researching on schema.org https://schema.org/IndividualProduct, but I found nothing.